run-tests.sh

```diff
@@ -1,120 +0,0 @@
-#!/bin/sh
```

Comment:
I don't see changes other than using `docker-compose`. I'm fine with dropping the script, but I'd like to understand the reason. Additionally, keep in mind that this does a couple of things that the rewrite does not; in particular it waits for the node to be able to reply before starting the tests.